app/src/main/java/org/sopt/and/ui/screen/home/composable/HomeScreen.kt

```diff
@@ -0,0 +1,316 @@
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.PagerState
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowForward
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import kotlinx.coroutines.delay
+import org.sopt.and.ui.screen.home.viewmodel.HomeViewModel
+import org.sopt.and.ui.screen.home.viewmodel.ImageOverviewViewState
+import org.sopt.and.ui.theme.darkGray1
+
+@Composable
+fun HomeScreen(
+    navController: NavHostController,
+    modifier: Modifier,
+    viewModel: HomeViewModel = hiltViewModel()
+) {
+    // 뷰모델에서 데이터를 가져옵니다.
+    val imageOverviews by viewModel.imageOverviews.observeAsState(emptyList())
+    val categories by viewModel.categories.observeAsState(emptyList())
+    val imageItemsType1 by viewModel.imageItemsType1.observeAsState(emptyList())
+    val imageItemsType2 by viewModel.imageItemsType2.observeAsState(emptyList())
+
+    val pagerState = rememberPagerState(
+        initialPage = 0, // 첫 번째 페이지부터 시작
+        initialPageOffsetFraction = 0f, // 페이지 오프셋 초기값 설정
+        pageCount = { imageOverviews.size } // 전체 페이지 수를 이미지 목록 크기로 설정
+    )
+    LazyColumn(
+        modifier = modifier
+            .background(darkGray1)
+    ) {
+        item {
+            HorizontalCategoryList(categories)
+        }
```

Comment:
여기에 stickyHeader를 적용해봐도 좋을 것 같아용
```suggestion
        stickyHeader {
            HorizontalCategoryList(categories)
        }
```

Reply:
감사합니다 참고해서 수정했습니다!